### PR TITLE
Underscore templating

### PR DESCRIPTION
**Pull request by cameronjacoby**

### CODE REVIEW

**Comment by cameronjacoby** [comment]
For week 2, day 2, module 1 (Tuesday morning)

File: w2_d2_1_underscore_templating/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+* A **template** is a document (or piece of code) that contains **parameters** which are dynamically replaced with data by the templating engine.
+
+* Thus far, we've been coding information (or data) directly into our HTML file. Eventually, the data on our HTML page will come from a server, and we'll need a way to dynamically display that data.
+
+* We'll use <a href="http://underscorejs.org/#template" target="_blank">Underscore's templating engine</a> to dynamically display data in our HTML, and the parameters will live inside `<%= %>` tags. We won't have data from a server yet, so for now, we'll store our data in an array.
+
+## Why use client-side templating?
+
+* Separate markup and from logic. Remember this?
```
Comment (janephilipps) [inline w2_d2_1_underscore_templating/README.md]: Confused by this sentence...is there a typo?

Comment (cameronjacoby) [inline w2_d2_1_underscore_templating/README.md]: Haha yeah, good catch! Should be "Separate markup from logic."

File: w2_d2_1_underscore_templating/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+2. Open your `index.html` in the browser to see if your test data displays on the page. If not, check the JS console to see what errors you're getting.
+
+3. Once you've successfully displayed your test data on the page, refactor your new To Do form to use the template when adding new tasks.
+
+## Stretch Challenges
+
+1. When a new task is created, make sure you are pushing it into the array of "test" data you set up in part 2 of the challenges to keep your model updated with your view.
+
+2. Implement a delete functionality to remove tasks from your To Do list. Think about the places you'll need to make updates (**Hint:** Model and View).
```
Comment (janephilipps) [inline w2_d2_1_underscore_templating/README.md]: will we have talked about model & view at this point? just checking so this hint doesn't confuse anyone.

Comment (cameronjacoby) [inline w2_d2_1_underscore_templating/README.md]: We'll be touching on it this afternoon, but I could definitely add in more info about that to make it more clear!

**Pull request merged**